lib/extensions/ar_migration.rb

```diff
@@ -0,0 +1,53 @@
+module MigrationSyncHelper
+  def self.migrations_column_present?
+    ActiveRecord::Base.connection.columns("miq_regions").detect { |c| c.name == "migrations_ran" }
+  end
+
+  class HelperARClass < ActiveRecord::Base; end
+  def self.restart_subscription(s)
+    c = HelperARClass.establish_connection.connection
```

Comment:
Do you need `HelperARClass`?  Can you use `ActiveRecord::Base` directly?

Reply:
I'm not sure. I think there's a single pool per AR class. Plus I didn't want to mess with the "real" connection pool here.

@jrafanie may have a better idea.